bpf/lib/common.h

```diff
@@ -78,7 +78,15 @@ enum {
 
 #define CILIUM_CALL_DROP_NOTIFY			1
 #define CILIUM_CALL_ERROR_NOTIFY		2
-#define CILIUM_CALL_SEND_ICMP6_ECHO_REPLY	3
+/*
```

Comment:
Personally, this is a bit much for a comment, as I can just git-blame to see what previous MACRO was there. 

Tho, I don't really feel to strongly about it, I think you could just say the bits about the gap, and the concerns around the backporting. 

This is a total nit, so feel free to ignore.